### DISCUSSION TITLE
Superadmin role

### DISCUSSION THREAD

**Discussion by ProjectVideoGame** [Pryaxis/TShock#2358] [Q&A]
aight, so-

I just set up my server right, then I follow instructions to add a super admin account, then I /off-ed the server, and when I turned it back on, I lose my super admin account, did I do something wrong?

edit: And how do I disable the L shortcut thingy (if you don't know, I have this problem where L will always open the command prompt window for the TerrariaServer.exe)

**Comment by hakusaro** [comment]
You have to login to the account you created. The answer I posted on [this discussion thread](https://github.com/Pryaxis/TShock/discussions/2325#discussioncomment-772279) more or less covers your options: you can either login to the account you created, or you can create a new account and give it the permissions you need.
